### DISCUSSION TITLE
Sandbox Last Refresh Date

### DISCUSSION THREAD

**Discussion by andruwaltringer** [burkybang/SuiteAdvanced-Feedback#71] [Ideas]
An idea could be to add the the most recent Sandbox Refresh Date next to where the NetSuite Account Number is currently being displayed.

**Comment by burkybang** [comment]
I researched possible ways to do this, but the date is not accessible from within the sandbox account. Still thinking about this one

**Comment by burkybang** [comment]
I released this feature in version 26.6.0

![image](https://user-images.githubusercontent.com/927886/235942444-4eced855-52dd-4d46-bf85-4187d9e6ed8e.png)
